src/components/NcAvatar/NcAvatar.vue

```diff
@@ -976,9 +976,9 @@ export default {
 		// - ⏹️ Best-fit: the status icon is as large as possible without exceeding the avatar box
 		// See PR for math explanation: PR #6004
 		--avatar-status-size-orbital: calc(var(--avatar-size) * (1 - 1 / sqrt(2)));
-		// Limit the status icon size to the status of a small clickable avatar
+		// Limit the status icon size to the status of a 50px avatar
 		// Ideally avatars with a smaller should not be used with the status icon at all
```

Comment:
This comment makes no sense anymore. Where do you use a 50px avatar?
I think most cases use default-clickable-area and thus it must work wit 34px